nixos/modules/system/boot/luksroot.nix

```diff
@@ -383,15 +484,9 @@ in
                 };
 
                 gracePeriod = mkOption {
-                  default = 2;
+                  default = 10;
                   type = types.int;
-                  description = "Time in seconds to wait before attempting to find the Yubikey.";
-                };
-
-                ramfsMountPoint = mkOption {
```

Comment:
Please add a release-note about the removal of `boot.initrd.luks.devices.<name?>.yubikey.ramfsMountPoint`, and the expected upgrade path for a user that previously used this option.